src/TConsole.cpp

```diff
@@ -772,14 +777,31 @@ void TConsole::closeEvent( QCloseEvent *event )
 
     if( profile_name != "default_host" && ! mUserAgreedToCloseConsole )
     {
-        ASK: int choice = QMessageBox::question( this, "Exiting Session: Question", "Do you want to save the profile "+profile_name, QMessageBox::Yes|QMessageBox::No|QMessageBox::Cancel );
+        QCheckBox *cb = new QCheckBox( tr("Save the Window Layout before closing this profile.") );
```

Comment:
In addition to language changes on the prompt, I also was thinking I should add something that makes the check box not show after the first save.  
After the first save of a window layout, the subsequent saves triggered by this prompt being checked will only save less and less window layout data.